packages/katello/rubygem-pulp_deb_client/rubygem-pulp_deb_client.spec

```diff
@@ -0,0 +1,91 @@
+# template: scl
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name pulp_deb_client
+%global gem_require_name %{gem_name}
+
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 2.3.0b1
```

Comment:
would you mind converting this to use the prerelease flag, similar to here: https://github.com/theforeman/foreman-packaging/blob/rpm/develop/packages/katello/rubygem-pulp_container_client/rubygem-pulp_container_client.spec#L8-L9

Because sadly gem versioning is the exact opposite of rpm versioning, i.e.   2.3.0b1 > 2.3.0 in rpm versioning, but its < in gem versioning.   Its commented out in that example because its using a ga version, but otherwise that is a good example

Reply:
or you could do the 0.1 similar to https://github.com/theforeman/foreman-packaging/pull/5130/files  

the prerelease flag is nice though :)

Reply:
Hope I got the usage right. Not sure why the test fails now.

Unfortunately, the prerelease-flag is not (yet) supported by my vim-script to create changelog-messages in spec-files :wink: